frontend/src/scenes/data-management/definition/definitionLogic.ts

```diff
@@ -93,6 +94,10 @@ export const definitionLogic = kea<definitionLogicType>([
                     } else {
                         await api.propertyDefinitions.delete({ propertyDefinitionId: values.definition.id })
                     }
+                    posthog.capture(`${values.singular} definition deleted`, {
```

Comment:
this should be in the backend - folks can hit the api above directly too

Reply:
Fair, moved